analyzers/tests/SonarAnalyzer.UnitTest/Rules/DeadStoresTest.cs

```diff
@@ -26,28 +26,33 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class DeadStoresTest
     {
+        private readonly VerifierBuilder builderSonarCfg = new VerifierBuilder()
```

Comment:
For simplicity. We now have something better to describe them except the `builder` fallback.
```suggestion
        private readonly VerifierBuilder sonarCfg = new VerifierBuilder()
```